resources/metadata/fortify.yaml

```diff
@@ -143,6 +148,23 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+        default: ./pom.xml
+      - name: buildDescriptorFile
```

Comment:
We should think of a better way to use different defaults than duplicating the parameters in the future.